spec/views/users/index.html.erb_spec.rb

```diff
@@ -1,5 +1,31 @@
 require 'rails_helper'
 
-RSpec.describe 'users/index.html.erb', type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'User Index Page', type: :feature do
+  it 'displays usernames of all users' do
+    visit users_path
+    User.all.each do |user|
+      expect(page).to have_content(user.name)
+    end
+  end
+
+  it 'displays profile pictures of all users' do
+    visit users_path
+    User.all.each do |user|
```

Comment:
- [ ] The below tests are failing, Please make sure that all tests pass without any errors. (I can see you attached a screenshot in the PR description for test results, If tests are passing on your end Please record a small video showing passing tests and also show that all tests are present in the project while testing).

![image](https://github.com/Anand8317/blog/assets/83803414/b8e24abe-f235-4dd4-a84c-18ba18b81165)

Reply:
Here is the video showing tests are passing on my end.
[Screen recording](https://share.icloud.com/photos/0e8VF8moEjQ37iRghsD5_Qhsg)